### PR TITLE
feat: remove a stat info from landing page

### LINKED ISSUE #8885: [FEATURE] Stats on Landing page | redundant information
### Description

The stats on the homepage shows duplicate information. Like "active users 2.8K from 2.8K"  I suggest only display the active users (28k) instead of showing the past ones.
Same for the other ones although it seems kind of wired links clicked xx from xx.

Hope this makes sense

### Screenshots

Old: 
image

New:
image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by rahulsiddarth**
Closes #8885 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by rizahmeds** [comment]
@rahulsiddarth Could you please add the screenshots?

**Review by accodes21 — COMMENTED** [review]
Looks good, thanks for contributing

File: components/statistics/BasicCards.js [hunk 1]
```diff
@@ -18,13 +18,6 @@ export default function BasicCards({ data }) {
               <dd className="mt-1 flex items-baseline justify-between sm:block lg:flex">
                 <div className="flex items-baseline text-2xl font-semibold text-secondary-medium dark:text-secondary-low">
                   {abbreviateNumber(item.current)}
-                  {item.total && (
-                    <span className="ml-2 text-sm font-medium text-primary-medium dark:text-primary-low-medium">
-                      <span title={item.total}>
-                        from {abbreviateNumber(item.total)}
-                      </span>
-                    </span>
-                  )}
```
Comment (eddiejaoude) [inline components/statistics/BasicCards.js:27]: This code needs to remain, this component is used in other places in the app

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you or your contribution 👍 

I have added inline comments

**Comment by eddiejaoude** [comment]
> Looks good, thanks for contributing

@accodes21 please do not approve pull requests without reviewing properly, this causes noise and confusing to the contributor and the maintainers

**Review by eddiejaoude — APPROVED** [review]
Looks good, thank you 👍

**Comment by accodes21** [comment]
> > Looks good, thanks for contributing
> 
> @accodes21 please do not approve pull requests without reviewing properly, this causes noise and confusing to the contributor and the maintainers

Extremely sorry for causing any confusion, will remember not to make such mistakes again

**Pull request merged**